core/metadata-3.2.md

```diff
@@ -120,6 +132,23 @@ using `ERR_KEYNOPERMISSION` with an asterisk (`*`) in the `<Key>` field.
 
 *Errors*: `ERR_KEYNOPERMISSION`
 
+### METADATA SYNC
+
+This subcommand requests the full synchronization of metadata associated with
+the given target. It is only to be used when `metadata-notify` is enabled
+by the client.
+
```

Comment:
I think this can be allowed even without `metadata-notify` to allow users to manually check metadata, even if their client doesn't enable notifications, just as they can with WHO.